.github/COMMIT_GUIDELINES.md

```diff
@@ -0,0 +1,11 @@
+# Commit Guidelines
+
+```
+<type><(scope)?>: <msg>
+```
+
+Where type is one of build, chore, ci, docs, feat, fix, perf, refactor, revert, style, or test.
```

Comment:
s/where/what

Reply:
What?

Reply:
Oh, sorry, I understand that phrasing now. maybe adding `'s to the tokens would help

Reply:
Where the type is one of
``` build, chore, ci, docs, feat, fix, perf, refactor, revert, style, or test.

Reply:
better ?